iOS/SonarKit.podspec

```diff
@@ -55,7 +57,12 @@ Pod::Spec.new do |spec|
     ss.dependency 'OpenSSL-Static', '1.0.2.c1'
     ss.compiler_flags = folly_compiler_flags
     ss.source_files = 'iOS/SonarKit/FBDefines/*.{h,cpp,m,mm}', 'iOS/SonarKit/CppBridge/*.{h,mm}', 'iOS/SonarKit/FBCxxUtils/*.{h,mm}', 'iOS/SonarKit/Utilities/**/*.{h,m}', 'iOS/SonarKit/*.{h,m,mm}'
-    ss.public_header_files = 'iOS/SonarKit/**/{SonarClient,SonarPlugin,SonarConnection,SonarResponder,SKMacros}.h'
+    # Including FBMacros here is hacky since it causes cocoapods to copy
+    # FBMacros.h twice, but it's necessary since the "FBMacros.h" is always
+    # included in SonarKit-umbrella.h, but the actual path is "FBDefines/FBMacros.h",
+    # as defined by the FBDefines subspec. Copying the file an extra time allows
+    # the path in the umbrella header to resolve correctly.
+    ss.public_header_files = 'iOS/FBDefines/FBMacros.h', 'iOS/SonarKit/**/{SonarClient,SonarPlugin,SonarConnection,SonarResponder,SKMacros}.h'
```

Comment:
Can't we get around this hack?

Reply:
If we leave FBMacros.h in its current state, then I don't think we can simultaneously support the import in SKMacros.h that looks like this:

```
#import <FBDefines/FBMacros.h>
```

at the same time as the import that is generated in the umbrella header that looks like this:

```
#import "FBMacros.h"
```

We also can't remove the umbrella header import, as far as I know, because FBMacros.h needs to be a public_header_file of the subspec since otherwise it won't be able to be found when importing SKMacros.h.

I was thinking that since FBMacros.h seems to only be used in SKMacros.h, we could simply move the contents of it into SKMacros.h - @priteshrnandgaonkar would this break anything internally?

Reply:
You can move the contents to `SKMacros.h`, it shouldn't cause internal things to break.